mode_manager_cli.py

```
@@ -0,0 +1,56 @@
+    while True:
+        cmd = input("\nEnter command (/mode <ask|build>, /edit <file>, /pending, /exit): ").strip()
+        if cmd.startswith("/mode"):
+            _, mode = cmd.split(maxsplit=1)
+            print(mm.set_mode(mode))
+        elif cmd.startswith("/edit"):
+            if not mm.is_build_mode():
+                print("Not in build mode. Use /mode build first.")
+                continue
+            _, file_path = cmd.split(maxsplit=1)
+            try:
+                orig = open(file_path, encoding="utf-8").read()
```

Ruff (0.12.2)

Use a context manager for opening files

(SIM115)